### DISCUSSION TITLE
Could not load the Qt platform plugin "xcb" in "/home/student/.local/lib/python3.9/site-packages/cv2/qt/plugins" even though it was found. This application failed to start because no Qt platform plugin could be initialized.

### DISCUSSION THREAD

**Discussion by Dragon2558** [wkentaro/labelme#1424] [Q&A / Help]
Hi everyone.

I'm trying to use label me for a Facial Recognition system and it keeps coming up with the same error:
[INFO   ] __init__:get_config:67- Loading config file from: /home/student/.labelmerc
QObject::moveToThread: Current thread (0x3bb6b3c0) is not the object's thread (0x3cc3f720).
Cannot move to target thread (0x3bb6b3c0)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/student/.local/lib/python3.9/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc.

Aborted

I've looked at the discussions on this error that others have posted over the years, sadly they did not work. I'm wondering if anyone has any solutions for this.

**Comment by Kimizhao** [comment]
ubuntu 22.04
i run this
sudo apt-get install '^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev
